lib/text/vtt_text_parser.js

```diff
@@ -843,6 +844,37 @@ shaka.text.VttTextParser = class {
 
     return (milliseconds / 1000) + seconds + (minutes * 60) + (hours * 3600);
   }
+
+  /**
+   * This method converts the HTML entities &amp;, &lt;, &gt;, &quot;, and
```

Comment:
Why don't you use a browser built-in for this?

Reply:
Oh, never mind, there isn't one.  I'm thinking of escaping URIs.

Reply:
I have found that it is slower and that it does not always work as I wish...

Reply:
See https://stackoverflow.com/questions/1912501/unescape-html-entities-in-javascript